io.github.yamanqd.antware.yml

```diff
@@ -0,0 +1,82 @@
+app-id: io.github.yamanqd.antware
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: run_antware.sh
+appdata-license: gpl-3.0
+
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.json
+  - name: SFML
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+        -DSFML_BUILD_EXAMPLES=0
+        -DSFML_BUILD_DOC=0
+        -DSFML_BUILD_AUDIO=1
+        -DSFML_BUILD_WINDOW=1
+    post-install:
+      - cp /app/pkgconfig/*.pc $FLATPAK_DEST/lib/pkgconfig/
+    sources:
+      - type: git
+        url: https://github.com/SFML/SFML.git
+        tag: 2.6.0
+        commit: 11b73743c42cf7ecd7c596ba83fdbf1150ffa96c
+        disable-shallow-clone: false
+  - name: ASSIMP
+    buildsystem: cmake
+    config-opts:
+      - -DASSIMP_NO_EXPORT=1
+        -DASSIMP_BUILD_ALL_EXPORTERS_BY_DEFAULT=0
+        -DASSIMP_BUILD_TESTS=0
+        -DASSIMP_INJECT_DEBUG_POSTFIX=0
+        -DASSIMP_INSTALL_PDB=0
+    sources:
+      - type: git
+        url: https://github.com/assimp/assimp.git
+        tag: v5.3.1
+        commit: 6a08c39e3a91ef385e76515cfad86aca4bfd57ff
+        disable-shallow-clone: false
+  - name: GLM
+    buildsystem: simple
+    build-commands:
+      -  cp -r ./glm $FLATPAK_DEST/include/
+    sources:
+      - type: git
+        url: https://github.com/g-truc/glm.git
+        tag: 0.9.9.8
+        commit: bf71a834948186f4097caa076cd2663c69a10e1e
+        disable-shallow-clone: false
+  - name: rapidjson
+    buildsystem: simple
+    build-commands:
+      - cp -r ./include/rapidjson $FLATPAK_DEST/include
+    sources:
+      - type: git
+        url: https://github.com/Tencent/rapidjson.git
+        tag: v1.1.0
+        commit: f54b0e47a08782a6131cc3d60f94d038fa6e0a51
+        disable-shallow-clone: false
+  - name: antware
```

Comment:
Appdata is not being installed

Reply:
How am I supposed to install use? install the metainfo file into...?

Reply:
With meson ideally eg. https://gitlab.gnome.org/World/amberol/-/blob/c8868d352ac364eafddfa7d46c3ee8474d114a16/data/meson.build#L28-40